grammar.js

```diff
@@ -541,13 +541,8 @@ module.exports = grammar({
     statement: $ => seq(
       choice(
         $._ddl_statement,
-        $._dml_read,
         $._dml_write,
-        seq(
-          '(',
-          $._dml_read,
-          ')',
-        ),
+        optional_parenthesis($._dml_read),
```

Comment:
okay it's really weird that you can wrap a `select` but not an `insert` or a `delete`

Reply:
@dmfay did you tested that? It is really astonishing how many things are not documented in postgres

Reply:
yeah even if you use a `returning` it refuses to parse any parenthesized write statement :exploding_head: